bin/config-example.json

```diff
@@ -0,0 +1,14 @@
+{
+  "files": [
+    "test/compare",
+    "test/declarative",
+    "test/imperative"
+  ],
+  "exclude": [],
+  "browser": {
+    "browsers": [
+      "Chrome"
+    ],
+    "logLevel": "disable"
```

Comment:
I think we should remove this or put it to 'debug' (to show that option exists) because having `disable` may confuse people (as there will be no output).

Reply:
There will be standard output as if we run it in node. To suppress output there is `quiet` like in `metatests` though it is not

Reply:
Also, `debug` will result in about 2000 lines output so that is not that should be a default option.